sklearn/ensemble/tests/test_weight_boosting.py

```diff
@@ -320,6 +319,65 @@ def test_sample_weight_missing():
     assert_raises(ValueError, clf.fit, X, y_regr)
 
 
+def test_sample_weight_adaboost_regressor():
+    """
+    AdaBoostRegressor should work without sample_weights in the base estimator
+
+    The random weighted sampling is done internally in the _boost method in
+    AdaBoostRegressor.
+    """
+    class DummyEstimator(BaseEstimator):
+
+        def fit(self, X, y):
+            pass
+
+        def predict(self, X):
+            return np.zeros(len(X))
+
+    boost = AdaBoostRegressor(DummyEstimator(), n_estimators=3)
+    boost.fit(X, y_regr)
+    assert_equal(len(boost.estimator_weights_), len(boost.estimator_errors_))
```

Comment:
With the adoption of pytest, we are phasing out use of test helpers `assert_equal`, `assert_true`, etc. Please use bare `assert` statements, e.g. `assert x == y`, `assert not x`, etc.

Reply:
I did not touch that test. It seems that I did because I removed and then readd the `test_sparse_classification` and `test_sparse_regression`. Do you want me to fix that even it is unrelated to the PR?